src/components/clusterWizard/ClusterWizardStep.tsx

```diff
@@ -64,7 +64,7 @@ const ClusterWizardStep: React.FC<ClusterWizardStepProps> = ({ cluster, footer,
         key="cluster-details"
         content={wizardStepNames['cluster-details']}
         isCurrent={currentStepId === 'cluster-details'}
-        isValid={() => !cluster || canNextClusterDetails({ cluster })}
+        isValid={() => !cluster?.validationsInfo || canNextClusterDetails({ cluster })}
```

Comment:
Not needed, see comments below.

Reply:
let's keep it for the first step, but remove for others since it is dead code there.

Reply:
Handling `canNextClusterDetails` within the create step would cause additional timing issues for that flow.

I hope I am not mistaken, but there should not be the case where the cluster is created but relevant validations errors are returned. I believe, the only valid flow during cluster create is either success or an HTTP error which is already handled.